backend/frontend/gulpfile.js

```diff
@@ -53,7 +53,7 @@ function build_node(){
 
   return gulp.src("src/**/*.js")
       .pipe(sourcemaps.init())
-      .pipe(babel({ optional: ['runtime'] }))
+      // .pipe(babel({ optional: ['runtime'] }))
```

Comment:
oh jeez, didn't notice I did that. Those two build targets are irrelevant tho. they are never used, and were just part of the gulpfile i copy-pasted from an old project